src/actions/users.js

```diff
@@ -610,6 +610,22 @@ export function getMe() {
     };
 }
 
+export function updateServiceTermsStatus(userId, serviceTermsId, accepted) {
+    return async (dispatch, getState) => {
+        await bindClientFunc(
```

Comment:
`dispatch(bindClientFunc(...))`

Reply:
If we don't use `await` here, the get call for the user object is completed before the `latest_service_terms_accepted` can be updated. It leads to getting stale data.